package.json

```diff
@@ -916,6 +922,12 @@
           "when": "view =~ /azure(ResourceGroups|Workspace|FocusView)/ && viewItem =~ /treeitem[.]database(?![a-z.\\/])/i && viewItem =~ /experience[.](mongocluster|mongodb)/i",
           "group": "2@2"
         },
+        {
+          "//": "[Database] Launch Cosmos Shell",
+          "command": "cosmosDB.launchCosmosShell",
+          "when": "view =~ /azure(ResourceGroups|Workspace|FocusView)/ && (viewItem =~ /treeitem[.]database(?![a-z.\\/])/i || viewItem =~ /treeitem[.]container(?![a-z.\\/])/i) && viewItem =~ /experience[.](table|cassandra|core)/i",
```

Comment:
Let's add this to root account nodes as well, and we should remove `table` and anything else besides `core`